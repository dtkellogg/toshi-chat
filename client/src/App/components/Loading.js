import React from 'react'
// import Loader from "react-loader-spinner";

export default function Loading() {
  return (
    <div className="loading" style={{'height': '100vh'}, {'width': '100vw'}}>
      <h1>Loading</h1>
    </div>
  )
}
