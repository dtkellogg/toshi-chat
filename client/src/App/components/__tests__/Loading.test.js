import Loading from '../Loading'


describe('tests for Loading Component', () => {
  test('renders loading text', () => {
    const wrapper = mount(<Loading />);
    expect(wrapper.text()).toMatch(/Loading.../)
  });
  
  test('matches snapshot', () => {
   const wrapper = shallow(<Loading />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
})