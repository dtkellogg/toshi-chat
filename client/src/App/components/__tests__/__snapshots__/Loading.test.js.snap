// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests for Loading Component matches snapshot 1`] = `
<div
  className="loading"
  style={
    Object {
      "width": "100vw",
    }
  }
>
  <h1>
    Loading...
  </h1>
</div>
`;
